app/src/angular/classes/wallet.js

```diff
@@ -64,7 +64,9 @@ class Wallet {
                 $rootScope.$broadcast('balance:change', 'eth', this.balanceEth, this.balanceInUsd);
                 if (readyToShowNotification) {
                     ElectronService.showNotification('ETH Balance Changed', 'New Balance: ' + this.balanceEth);
-                    // TODO - scan transactions
+                    setTimeout(() => {
+                        Web3Service.syncWalletActivityByETH();
```

Comment:
use **$timeout** instead of **setTimeout** or much better to listen **balance:change** event